include/Target/Btor/BtorToBtorIRTranslation.h

```diff
@@ -201,7 +208,7 @@ class Deserialize {
     }
     auto res = m_builder.create<btor::ConstantOp>(
                         FileLineColLoc::get(m_sourceFile, lineId, 0),
-                        type, m_builder.getIntegerAttr(type, value));
```

Comment:
I made this comment in the td file but it's worth having a Btor_BitVector Attribute

Reply:
Custom btor bitvec attributes have been implemented